src/applications/vaos/components/NoValidVAFacilities.jsx

```diff
@@ -1,28 +1,88 @@
+  const phone = attributes?.phone;
+  const hours = attributes?.hours;
 
-export default function NoValidVAFacilities({ systemId }) {
   return (
     <div aria-atomic="true" aria-live="assertive">
       <AlertBox
         status="warning"
-        headline="Sorry, we couldn't find any locations for you"
+        headline={`We’re sorry. This location doesn’t allow online scheduling for ${typeOfCare} appointments`}
       >
-        Some types of care are not available at certain VA locations and not all
-        VA locations have enabled online scheduling. You can choose a different
-        VA health system or{' '}
-        <a
-          href={`/find-locations/facility/vha_${systemId}`}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          contact the VA medical center you chose
-        </a>{' '}
-        to get help with the care you need.
         <p>
-          <Link to="/">Go back to VA Online Scheduling home.</Link>
+          You can’t schedule this type of appointment online. Please call the
+          medical center for more information.
         </p>
+        {systemDetails ? (
+          <div className="vads-u-padding-left--2 vads-u-border-left--4px vads-u-border-color--primary">
+            <span className="vads-u-font-weight--bold">{attributes?.name}</span>
+            <br />
+            <span>{address?.address1}</span>
+            <br />
+            {!!address?.address2 && (
+              <>
+                <span>{address?.address2}</span>
+                <br />
+              </>
+            )}
+            <span>
+              {address?.city}, {address?.state} {address?.zip}
+            </span>
+            <br />
+            <LocationDirectionsLink location={systemDetails} hideIcon />
+            <div className="vads-u-display--flex vads-u-margin-top--2">
+              <span className="vads-u-font-weight--bold">Hours:</span>
+              <ul className="usa-unstyled-list vads-u-margin-top--0">
+                {hours &&
+                  Object.keys(hours).map((day, index) => (
+                    <li key={`day-${index}`}>{titleCase(day)}</li>
+                  ))}
+              </ul>
+              <ul className="usa-unstyled-list vads-u-margin-top--0">
+                {hours &&
+                  Object.keys(hours).map((day, index) => (
+                    <li key={`hours-${index}`}>{hours[day]}</li>
+                  ))}
+              </ul>
+            </div>
+            <p>
+              <span className="vads-u-font-weight--bold">Main phone: </span>
+              <a href={`tel:${phone?.main?.replace('-', '')}`}>{phone?.main}</a>
+            </p>
+            {!!phone?.mentalHealthClinic && (
+              <p>
+                <span className="vads-u-font-weight--bold">
+                  Mental health phone:{' '}
+                </span>
+                <a href={`tel:${phone?.mentalHealthClinic?.replace('-', '')}`}>
```

Comment:
Could probably remove some of the `?` here, since we know there's a mental health clinic phone if we're running this code.